Plugins/GRpluginStands.txt

```diff
@@ -1167,115 +1167,215 @@ USE:C
 CODE:E
 
 //STAND 313
-STAND:ENGM:313:N060.11.55.530:E011.04.34.750:20
+STAND:ENGM:313
+COORD:N060.11.56.214:E011.04.37.733
+COORD:N060.11.55.301:E011.04.37.898
+COORD:N060.11.55.106:E011.04.33.037
+COORD:N060.11.56.029:E011.04.32.926
+COORD:N060.11.56.214:E011.04.37.733
```

Comment:
Need to test the position of this polygon somewhat. Either it's a bit off or the label should be moved.